.travis.yml

```diff
@@ -3,14 +3,8 @@ os:
 language: node_js
 node_js:
   - "node"
-  - "lts/*"
-  - "4"
-
-before_script:
-  # Fixes an annoying bug in ESLint on Node 4
-  # https://github.com/babel/babel-eslint/issues/558
-  # TODO: Remove when Node 4 is not supported anymore
-  - npm install estraverse
+  - "8"
+  - "6"
```

Comment:
Why not `lts/*`? 😁

Reply:
I had that first 😆 
Might be my personal preference, but I refer to it as "Node.js 8" and not "Node.js Carbon". We can change that ofc, but I guess more people have to look up codenames whereas version is pretty straight forward to understand 😄

Reply:
Ah, sorry. I guess I overinterpreted your question.
`lts/*` would only cover Node.js 8 currently and `""node", lts/*", "6"` just looked wrong 😆

Reply:
Makes sense 👍